src/ui/asset-list/asset-list.lite.tsx

```diff
@@ -65,6 +65,8 @@ export default function AssetList(props: AssetListProps) {
                   showWithdraw={item.showWithdraw}
                   onDeposit={() => item?.onDeposit()}
                   onWithdraw={() => item?.onWithdraw()}
+                  withdrawLabel={item.withdrawLabel ?? AssetListItem.defaultProps.withdrawLabel}
```

Comment:
looks good, but I don't think you need the last bit `?? AssetListItem.defaultProps.withdrawLabel`, `AssetListItem` already has default props for those :) remove those and I'll get it published

Reply:
done. thanks for the review